### ISSUE TITLE
Videos are not working on Safari 11.0.3

### ISSUE THREAD

**Issue by mrpau-richard** [learningequality/kolibri#5109]
### Observed behavior
I can't play videos on Safari 11.0.3

![screen shot 2019-02-20 at 5 28 16 pm](https://user-images.githubusercontent.com/8663934/53081352-a63c0180-3535-11e9-8b58-8c5aad3d67a6.png)

### Errors and logs

```
[Error] VIDEOJS: (4)
"ERROR:"
"(CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED)"
"The media could not be loaded, either because the server or network failed or because the format is not supported."
MediaError
	(anonymous function) (media_player_module-0.12.0b7.js:1:209326)
	error (media_player_module-0.12.0b7.js:1:435778)
	handleTechError_ (media_player_module-0.12.0b7.js:1:423419)
	dispatcher (media_player_module-0.12.0b7.js:1:222733)
[Error] Unhandled Promise Rejection: [object DOMError]
```

### Context

Tell us about your environment, including:
 * Kolibri version: 0.12-beta7
 * Operating system: macOS 11.13.3
 * Browser: Safari 11.0.3

**Comment by mrpau-julius** [comment]
Also on Windows 10 chrome version 72.0.3636.109(official build)(64-bit), the videos are not working.
![video not working on chrome](https://user-images.githubusercontent.com/38421180/53083912-bd312280-353a-11e9-9298-18f1ce85e179.png)

**Comment by indirectlylit** [comment]
does this happen with all videos or just some?

**Comment by indirectlylit** [comment]
might relate to cherrypy config

**Comment by mrpau-richard** [comment]
> does this happen with all videos or just some?

On Safari all videos can't play.
I also tested those videos on Firefox and it play properly with no console errors.

**Comment by indirectlylit** [comment]
@mrpau-richard mind verifying this fix using the build from #5141?
